en/docs/concepts/compliance/gdpr.md

```diff
@@ -0,0 +1,226 @@
+    purposes, and time-limits and what data stored can be made
+    transparent to users via the WSO2 IS **My Account** application.  
+   
+-   **The right of access -** Via the WSO2 IS **My Account** application, users
+    can access and review what personal data is stored in the
+    processing organization.  
+
+-   **The right to rectification -** Individuals can rectify incorrect
+    data on their user profiles by themselves by logging in to the **My Account** application.
+
+-   **The right to restrict processing -** Individuals can make
+    restrictions on their user profiles by themselves by logging into
+    the **My Account** application. Generally, this is done by revoking
+    an already given consent but can be extended to other usages as
+    well.  
+
+-   **The right to be forgotten -** This is one of the most important individual rights defined in GDPR. In
+simple terms, an individual can request to completely remove their
+personal data from the processing organizations. According to GDPR,
+unless there is a clear and valid legal background, processing
+organizations should fulfill such *forget me* requests.
+
+WSO2 IS provides out of a box privacy toolkit to remove all identify
+data from related databases and log files. This toolkit can be run
+manually by organization administrators or can be automated so that
+whenever a user profile gets deleted from the system, all the related
+PII data gets removed from the system.
+
+By considering performance overhead and automation flexibility, this
+privacy toolkit is run separately from WSO2 IS runtime. For older
+versions of WSO2 IS, it is required to download WSO2 Privacy
+Toolkit from
+**[here](https://github.com/wso2/identity-anonymization-tool)**
+separately.
+
+When it comes to **Right to be forgotten**, WSO2 IS supports the following
+features.
+
+-   Delete the user by “Identity Admin” of the tenant. This will remove
+    the user from any underlying “Read/Write” userstore (JDBC/LDAP/AD).
+-   Anonymize any retained traces of the user activity.  
+    -   Log Files
+    -   Analytics data, related to login, session, key validation, etc.
+    -   Key/token data held at the database layer.
+-   Delete any unwanted data retained in the database(due to performance
+    reasons)
+    -   Token(s) issued,
+    -   Password History information.
+
+For more information on the topic, see [Removing References to Deleted User Identities](TO-DO: insert-extend-link). 
+
+-   **The right for notification obligation -** The **My Account** application can be extended to act as the notification center for
+    individuals.  
+
+-   **The right to data portability -** Individuals can download their
+    user profile in a structured, commonly used and machine-readable
+    JSON document format through the **My Account** application. Ability to download individual’s user profile as a structured, commonly
+used in machine-readable format is a requirement of GDPR. In WSO2 IS,
```

Comment:
Ability to download individual’s user profile as a structured, commonly
used in machine-readable format is a requirement of GDPR.

seems unnecessary to repeat this sentence..maybe we can say "which is a requirement of GDPR" at the end of the previous sentence instead